src/app/commitment/testing/fake-commitment.service.ts

```diff
@@ -0,0 +1,54 @@
+import {TestHelper} from '../../../testing/test.helper'
```

Comment:
There needs to be spacing after `{` and before `}`. Can you please fix this in your files?
Example:
```ts
import { TestHelper } from ...
```